luatest/server.lua

```diff
@@ -117,6 +117,7 @@ function Server:new(object, extra)
                     if not object.tests[t.name] then
                         object.tests[t.name] = t
                         t.servers[object.id] = object
+                        log.verbose('Server %s used in %s test', object.alias, t.name)
```

Comment:
What about adding `(pid: %d)`?

Reply:
I would not like to add this because there may be a situation where `self.process` does not exist:

```lua
g.before_all(function()
    g.server = server:new()
end)
g.test_foo = function()
    g.server.<something>(...) -- server used in `test_foo`
    -- but server does not started -> self.process == nil -> self.process.id -> error
```

Reply:
OK, got it.